cmd/terramate/cli/cli.go

```diff
@@ -1331,6 +1332,11 @@ func (c *cli) printStacks() {
 
 	c.gitFileSafeguards(false)
 
+	if c.parsedArgs.List.RunOrder {
+		c.printRunOrder(true)
+		return
+	}
```

Comment:
AFAIK the `list --run-order` must use the `list` output but sorted by the topological order. The existent run-order will be kept there as before but made hidden.

It just an additional flag to list, then you can even do `terramate list --changed --run-order --why`.

Reply:
That makes sense, will fix.